Assets/MixedRealityToolkit.Services/InputSimulation/Editor/InputSimulationService.cs

```diff
@@ -208,6 +208,8 @@ public bool CheckCapability(MixedRealityCapability capability)
         /// <inheritdoc />
         public override void Initialize()
         {
+            base.Initialize();
```

Comment:
Interesting. I wonder if this can break input simulation? Would be good to check that input simulation is still working after adding the base calls.

Reply:
Why would this break input simulation? The hands interaction example scene worked for me in editor which uses this service for the input.